pkg/registry/pod/etcd/etcd_test.go

```diff
@@ -37,10 +38,11 @@ import (
 	"k8s.io/kubernetes/pkg/util"
 )
 
-func newStorage(t *testing.T) (*REST, *BindingREST, *StatusREST, *tools.FakeEtcdClient) {
+func newStorage(t *testing.T) (*REST, *BindingREST, *StatusREST, *tools.FakeEtcdClient, PodStorage) {
 	etcdStorage, fakeClient := registrytest.NewEtcdStorage(t, "")
-	storage := NewStorage(etcdStorage, generic.UndecoratedStorage, nil, nil)
-	return storage.Pod, storage.Binding, storage.Status, fakeClient
+	nodeREST, _ := nodeetcd.NewREST(etcdStorage, generic.UndecoratedStorage, nil, nil)
```

Comment:
not a fan of including another registry impl in this test... why can't we continue passing nil for the ConnectionInfoGetter?